README.md

```diff
@@ -20,3 +20,25 @@ For other issues, please use the following repos:
 - For class library and framework functioning issues, file on [dotnet/corefx](https://github.com/dotnet/corefx) repo
 - For runtime issues, file on [dotnet/coreclr](https://github.com/dotnet/coreclr) issues
 
+Build Status
+------------
+
+|Ubuntu 14.04 |Ubuntu 16.04 |Debian 8.2 |Windows x64 |Windows x86 |Mac OS X |CentOS 7.1 |RHEL 7.2 |Fedora 23 |OpenSUSE 13.2 |
```

Comment:
We are going to have to keep these in sync across repos too? This will get out of control soon.